docs/source/contributing/git.rst

```diff
@@ -65,6 +65,33 @@ covers setting it up. To access repositories within an enterprise environment
 
 .. _pull-request:
 
+Useful git commands
+-------------------
+
+While git has many commands, for most things you will find the following list of
+commands sufficient. More detail on each of these commands can be found in the
+`git reference documentation`_.
+
+.. csv-table::
+    :header: "Command", "Description"
+    :widths: auto
+
+    "``git status``", "Status of repository, showing changed files, etc."
+    "``git clone <repo-url>``", "Download code"
+    "``git add <filename>``", "Add a modified or new file to your next commit"
+    "``git branch``", "List local branches"
+    "``git switch <branch>``", "Switch to an existing branch"
+    "``git switch -c <branch>``", "Create a new branch"
```

Comment:
should we say here that it creates a local branch in your local repository? This then needs to get pushed into the remote?